ecosystem/typescript/sdk/src/faucet_client.ts

```diff
@@ -17,9 +17,20 @@ export class FaucetClient extends AptosClient {
    * @param config An optional config for inner axios instance
    * Detailed config description: {@link https://github.com/axios/axios#request-config}
    */
-  constructor(nodeUrl: string, faucetUrl: string, config?: AptosClientConfig) {
+  constructor(nodeUrl: string, faucetUrl: string, config?: OpenAPIConfig) {
     super(nodeUrl, config);
-    this.faucetUrl = faucetUrl;
+    // Build a requester configured to talk to the faucet.
+    this.faucetRequester = new AxiosHttpRequest({
+      BASE: faucetUrl,
+      VERSION: config?.VERSION ?? "0.1.0",
```

Comment:
Don't do this. "??" only checks nulls. "?" might give you undefined.

Reply:
What should I do instead? For context, this is what the generated client does as well.

Reply:
I need to brush up my understanding with "??". It checks both null and undefined. https://www.typescriptlang.org/play?#code/LAKAxg9gdgzhA2BTAdPCBzAFAVygE0QDMBLKRPAAgH4qKAiAC0XjToEpQg